
From: Adrian Bunk <bunk@stusta.de>

The placement of CPUSETS somewhere in the middle of the EMBEDDED options
breaks the EMBEDDED submenu (at least in menuconfig).

The patch below fixes this by simply moving CPUSETS above EMBEDDED.

Signed-off-by: Adrian Bunk <bunk@stusta.de>
Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/init/Kconfig |   21 ++++++++++-----------
 1 files changed, 10 insertions(+), 11 deletions(-)

diff -puN init/Kconfig~cpusets-move-cpusets-above-embedded init/Kconfig
--- 25/init/Kconfig~cpusets-move-cpusets-above-embedded	2005-01-10 21:15:15.486099336 -0800
+++ 25-akpm/init/Kconfig	2005-01-10 21:15:15.489098880 -0800
@@ -237,6 +237,16 @@ config IKCONFIG_PROC
 	  This option enables access to the kernel configuration file
 	  through /proc/config.gz.
 
+config CPUSETS
+	bool "Cpuset support"
+	depends on SMP
+	help
+	  This options will let you create and manage CPUSET's which
+	  allow dynamically partitioning a system into sets of CPUs and
+	  Memory Nodes and assigning tasks to run only within those sets.
+	  This is primarily useful on large SMP or NUMA systems.
+
+	  Say N if unsure.
 
 menuconfig EMBEDDED
 	bool "Configure standard kernel features (for small systems)"
@@ -291,17 +301,6 @@ config EPOLL
 	  Disabling this option will cause the kernel to be built without
 	  support for epoll family of system calls.
 
-config CPUSETS
-	bool "Cpuset support"
-	depends on SMP
-	help
-	  This options will let you create and manage CPUSET's which
-	  allow dynamically partitioning a system into sets of CPUs and
-	  Memory Nodes and assigning tasks to run only within those sets.
-	  This is primarily useful on large SMP or NUMA systems.
-
-	  Say N if unsure.
-
 config CC_OPTIMIZE_FOR_SIZE
 	bool "Optimize for size" if EMBEDDED
 	default y if ARM || H8300
_
